lib/guest_installation_and_configuration_base.pm

```diff
@@ -919,7 +924,7 @@ sub config_guest_installation_method {
     $self->reveal_myself;
     $self->config_guest_params(@_) if (scalar(@_) gt 0);
     if ($self->{guest_installation_method} eq 'location') {
-        $self->{guest_installation_media} =~ s/Build12345/Build$self->{guest_build}/g if ($self->{guest_build} ne 'gm');
+        $self->{guest_installation_media} =~ s/12345/$self->{guest_build}/g if ($self->{guest_build} ne 'gm');   #Use 12345 rather than Build12345 in light of openSUSE Tumbleweed build
```

Comment:
Can sle15sp4 daily build survive with this change too?

Reply:
supposed to be.

Reply:
the media in guest param file for TW is */openSUSE-Tumbleweed-DVD-x86_64-Snapshot12345. for SLE is */SLE-15-SP3-Full-x86_64-Build12345-Media1. So just replace 12345.